sdk-extensions/autoconfigure/src/test/java/io/opentelemetry/sdk/autoconfigure/LogEmitterProviderConfigurationTest.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.autoconfigure;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.metrics.MeterProvider;
+import io.opentelemetry.sdk.logs.LogProcessor;
+import io.opentelemetry.sdk.logs.SdkLogEmitterProvider;
+import io.opentelemetry.sdk.resources.Resource;
+import java.util.Collections;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+
+class LogEmitterProviderConfigurationTest {
+
+  @Test
+  void configureLogEmitterProvider() {
+    Map<String, String> properties = Collections.singletonMap("otel.logs.exporter", "none");
+
+    Resource resource = Resource.create(Attributes.builder().put("cat", "meow").build());
+    // We don't have any exporters on classpath for this test so check no-op case. Exporter cases
+    // are verified in other test sets like testFullConfig.
+    SdkLogEmitterProvider logEmitterProvider =
+        LogEmitterProviderConfiguration.configureLogEmitterProvider(
+            resource, DefaultConfigProperties.createForTest(properties), MeterProvider.noop());
+    try {
+      assertThat(logEmitterProvider)
+          .extracting("sharedState")
+          .satisfies(
+              sharedState -> {
+                assertThat(sharedState).extracting("resource").isEqualTo(resource);
+                assertThat(sharedState)
+                    .extracting("logProcessor")
+                    .isEqualTo(LogProcessor.composite());
```

Comment:
I'm not sure this is really verifying that we have a no-op. Just that our processor is a composite. Can we make a stronger assertion here?

Reply:
It is verifying it, but in a cheating way:`LogProcessor.composite()` returns a `NoopLogProcessor`. Its doing it this way because `NoopLogProcessor` is package private and inaccessible to both `LogEmitterProviderConfiguration` and the test code. Would have to do something like this to validate its a noop:
```
      .isInstanceOfSatisfying(
          LogProcessor.class,
          logProcessor ->
              assertThat(logProcessor.getClass().getName())
                  .isEqualTo("io.opentelemetry.sdk.logs.NoopLogProcessor"));
```

Arguably, verifying that it's a `NoopLogProcessor` is beyond the scope of the test. `LogEmitterProviderConfiguration` doesn't do anything except add 0 or more `LogProcessor`s to the `SdkLogEmitterProviderBuilder`. Its not till `LogEmitterSharedState` is constructed in the `SdkLogEmitterProvider` constructor that the list of configured log processors is translated to a single log processor via `LogProcessor.composite(LogProcessor...)`. 

By having the test verify that the log processor is equal `LogProcessor.composite()` (with zero log processors), we verify that `LogEmitterProviderConfiguration` did not add any.

Reply:
I totally missed that `composite` took varargs processors. 👍🏽